onnxruntime/python/tools/quantization/registry.py

```diff
@@ -79,7 +79,6 @@
     "MatMul": QDQMatMul,
     "Split": QDQSplit,
     "Gather": QDQGather,
-    "Softmax": QDQSoftmax,
```

Comment:
It is not needed anymore or it was not used or...

Reply:
It is not needed anymore (QDQ softmax is now handled by the QDQBaseOperator class).

This class was only forcing Softmax to range from 0.0 to 1.0. The same can be accomplished by adjusting the calibration data range as is currently done for Clip and Relu:
https://github.com/microsoft/onnxruntime/blob/caf9c7dd4ecfd30eea4dbe48f6e82e2b97344bba/onnxruntime/python/tools/quantization/base_quantizer.py#L455-L476

This adjustment is only enabled for QDQ quantizer to keep the same behavior as before this PR.

Reply:
Additionally, the QDQSoftmax class was the only place where the `set_quant_scale_zp()` and `find_quant_scale_zp()` functions were used. The removal of the QDQSoftmax class also allows this PR to remove these functions, which would have otherwise required a non-trivial update.